examples/cluster-configs/trn1-16-nodes-pcluster.md

```diff
@@ -15,6 +15,13 @@ HeadNode:
   LocalStorage:
     RootVolume:
       Size: 1024
+  CustomActions:
```

Comment:
can we put a note on the bottom of the YAML regarding this to let customers know that the installation is done after launch automatically ?

Reply:
See #3 below. Installation is done "during" creation. It won't say creation completed until installation on headnode and any static compute node is done.

"During the cluster creation process, post-install actions now takes place automatically via `CustomActions` indicated in `launch.yaml` to configure the head node and any static compute nodes (`MinCount` > 0). `CustomActions` will install Neuron drivers and runtime, EFA drivers, and Neuron tools. "